share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -107,39 +56,38 @@ class Foo < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}", "--disable-debug", "--disable-dependency-tracking"
-#   system "cmake", ".", *std_cmake_args
+    # system "cmake", ".", *std_cmake_args
```

Comment:
I don't mind either way, actually. I'll just update this to whatever `brew create` does.